### ISSUE TITLE
Build lxml whl file for python 3.10

### ISSUE THREAD

**Issue by Storkycold** [chaquo/chaquopy#1258]
#### Chaquopy version

https://chaquo.com/chaquopy/chaquopy-version-15-0-1/

chaquopy: 15.0.1 

#### Relevant parts of your build.gradle file

chaquopy: 15.0.1 
python: 3.10 
ndk: 27.0.12077973

#### Describe your issue

I tried to make lxml package whl file in Linux, targeted to Android env. `python 3.10` is necessary. Downgrading python version to 3.9 or 3.11 is not our consideration. 

ex) `./build-wheel.py --python 3.10 --abi arm64-v8a lxml`

### prerequisite 

- downloaded chaquopy-libxml2, chaquopy-libxslt from https://chaquo.com/pypi-13.1/, and saved it on `./server/pypi/dist`

    - chaquopy-libxml2 2.9.8
    - chaquopy-libxslt 1.1.32 

- set `ANDROID_HOME` as env_variable, which downloaded from https://github.com/chaquo/chaquopy/tree/master/server/pypi#setup instruction. 

`export ANDROID_HOME=/home/storkycold/dev/android-sdk`

In this case, after implementing `./build-wheel.py --python 3.10 --abi arm64-v8a lxml`, this error persists. 

```
Chaquopy: ignored invalid -I directory: '/home/storkycold/include/libxml2'
Chaquopy: ignored invalid -I directory: '/usr/include/libxml2'
Chaquopy: ignored invalid -I directory: '/usr/include/python3.10'
In file included from src/lxml/etree.c:772:
src/lxml/includes/etree_defs.h:14:10: fatal error: 'libxml/xmlversion.h' file not found
#include "libxml/xmlversion.h"
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.
Compile failed: command '/home/storkycold/dev/chaquopy/server/pypi/packages/lxml/build/4.6.5/cp310-cp310-android_24_arm64_v8a/wrappers/aarch64-linux-android24-clang' failed with exit code 1
creating tmp
cc -I/home/storkycold/include/libxml2 -I/usr/include/libxml2 -I/usr/include/libxml2 -c /tmp/xmlXPathInitxt3zqtjp.c -o tmp/xmlXPathInitxt3zqtjp.o
cc tmp/xmlXPathInitxt3zqtjp.o -L/home/storkycold/lib -lxml2 -o a.out
error: command '/home/storkycold/dev/chaquopy/server/pypi/packages/lxml/build/4.6.5/cp310-cp310-android_24_arm64_v8a/wrappers/aarch64-linux-android24-clang' failed with exit code 1
build-wheel: Error: Backend subprocess exited when trying to invoke build_wheel
```

In linux, this header file does successfully show up. 

```
$ ls /home/storkycold/android_libs/libxml2/include/libxml2/libxml/xmlversion.h

/home/storkycold/android_libs/libxml2/include/libxml2/libxml/xmlversion.h
```

Stuck in this situation, would it be possible to ask to build `lxml package wheel file` that is available for `python 3.10`, targeted to Android env? 

Providing us critical solution would be an great option for us too. 

Currently, lxml package is only available for `python 3.8` (ref: https://chaquo.com/pypi-13.1/lxml/)

<img width="450" alt="image" src="https://github.com/user-attachments/assets/78c426f1-40fe-4156-b51a-6b8cd4c60bdb">

Deeply appreciate for your consideration. 

Thanks.

**Comment by mhsmith** [comment]
I've tried the process you describe, and it works fine for me. Please post the full build log. And if you've made any changes to the repository, please post them as well.

Related: #816

**Comment by Storkycold** [comment]
Sorry for the omissions. 

> I've tried the process you describe, and it works fine for me.  

Does this mean that building lxml wheel file that is, 

- available for `python 3.10`
- chaquopy_libxml2-2.9.8 
- chaquopy-libxslt 1.1.32
- lxml version: 4.6.3? 

Would it be possible for you to ask for the exact mandatory process to `build lxml wheel file for python 3.10`? 

> /home/storkycold/include/libxml2 

For this one, I downloaded directly in linux and tried to use header files from here. But I guess its not necessary, I would like to try with clean slate with your guidance. 

```
wget http://xmlsoft.org/sources/libxml2-2.9.12.tar.gz
tar -xzvf libxml2-2.9.12.tar.gz
cd libxml2-2.9.12
```

Thanks.

**Comment by mhsmith** [comment]
Remove /home/storkycold/lib and /home/storkycold/include (or at least the libxml2 things in those directories), and then run the build-wheel command again.

And if you've set any environment variables while working on this (other than ANDROID_HOME), then `unset` them.

**Comment by mhsmith** [comment]
`lxml` version 5.3.0 is now available for Python 3.9 - 3.13.

**Comment by anonyein** [comment]
> `lxml` version 5.3.0 is now available for Python 3.9 - 3.13.

could Python 3.8 be added?
